clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -163,6 +165,168 @@ public static int readUnsignedIntLE(byte[] buffer, int offset) {
     }
 
     /**
+     * Read an unsigned integer stored in variable-length format.
```

Comment:
Yes, we've linked to the snappy encoding somewhere, but we should be clearer if we've copied the code.